Library/Formula/packer-community-plugins.rb

```diff
@@ -0,0 +1,45 @@
+class PackerCommunityPlugins < Formula
+  homepage "https://github.com/packer-community/packer-windows-plugins"
+  version "0.0.1"
+
+  if Hardware.is_64_bit?
+    url "https://github.com/packer-community/packer-windows-plugins/releases/download/pre-release/darwin_amd64.zip"
```

Comment:
Is this software stable yet? It looks like `pre-release` is the only version so far. Where did you get the 0.0.1 number?